asllib/builder.ml

```diff
@@ -61,6 +61,9 @@ let from_lexbuf ast_type version (lexbuf : lexbuf) =
       and lexer0 = Lexer0.token () in
       try parse lexer0 lexbuf with Parser0.Error -> cannot_parse lexbuf)
 
+let from_lexbuf' ast_type version lexbuf () =
```

Comment:
If it's not too much work, could you please describe what this commit is all about?